website/content/v1.8/talos-guides/install/cloud-platforms/hetzner.md

```diff
@@ -255,6 +279,13 @@ This can be found by issuing:
 hcloud server list | grep talos-control-plane
 ```
 
+Before the bootstrap process, you need to apply controlplane config to the first node:
```

Comment:
the configuration is poassed via `--user-data-from-file`, so looks like something else is going wrong here

Reply:
ah, you're using metal- image instead of hcloud one (you should use on from the Image Factory), and then these steps are not needed.

Reply:
Ah, probably you are right about the bootstrap process.

However, the part about additional variants of image upload is still present.

Reply:
Oh, I understand my mistake. I will make changes to my suggestion of changes to docs.